docs/_ldio/ldio-adapters/index.md

```diff
@@ -8,6 +8,13 @@ nav_order: 4
 
 # Linked Data Orchestrator Adapters
 
-The LDI Core module contains the components maintained by the VSDS team in order to accommodate the onboarding of LDES onboarders.
+Adapters are be used in conjunction with the LDI Input. They will transform the provided content into and internal Linked Data model and sends it down the pipeline for further processing.
 
-Each component can be wrapped in a desired implementation framework (LDI-orchestrator, NiFi, ...) to be used.
+## Overview
+
+| Adapter             | Description                                                                              | Advantages                                                     | Disadvantages                                                               |
+|:--------------------|:-----------------------------------------------------------------------------------------|:---------------------------------------------------------------|:----------------------------------------------------------------------------|
+| _JSON to JSON-LD_   | Receives JSON messages and adds linked data context to transform the messages to JSON-LD | Readability of JSON-LD format Easy to set up (plug in context) | Only works with JSON as input Slower performance when deserializing model   |
+| _NGSIv2 to NGSI-LD_ | Converts NGSIv2 to an NGSI-LD model                                                      |                                                                | Only works with NGSIv2 as input Slower performance when deserializing model |
+| _RDF_               | Takes in an RDF string and converts it into an internal linked data model                | Most basic adapter Easy set-up: no configuration needed        | Only works with RDF as input Only supports valid RDF MIME types             |
+| _RML_               | Transform a non-linked data object (JSON/CSV/XML) to RDF object                          | Most powerful adapter Can convert multiple input objects       | Most complex to create a mapping (external tooling can help)                |
```

Comment:
Most complex to create a mapping (external tooling can help) -> requires the knowledge of RML to map ?